.github/workflows/test_release.yml

```diff
@@ -30,33 +30,25 @@ jobs:
       fail-fast: false
       matrix:
 
-        target:
-          - {
-            host: windows-latest,
-            os: windows,
-            build: dmd.master.windows.7z
-          }
-          - {
-            host: macos-latest,
-            os: osx,
-            build: dmd.master.osx.tar.xz
-          }
-          - {
-            host: ubuntu-latest,
-            os: linux,
-            build: dmd.master.linux.tar.xz
-          }
-          - {
-            host: macos-10.15,
-            os: freebsd,
-            build: dmd.master.freebsd-64.tar.xz
-          }
-
-    runs-on: ${{ matrix.target.host }}
+        include:
+          - host: windows-latest
+            os: windows
+            build: dmd.${{ github.base_ref || github.ref_name }}.windows.7z
+          - host: macos-latest
+            os: osx
+            build: dmd.${{ github.base_ref || github.ref_name }}.osx.tar.xz
+          - host: ubuntu-latest
+            os: linux
+            build: dmd.${{ github.base_ref || github.ref_name }}.linux.tar.xz
+          - host: macos-10.15
+            os: freebsd
+            build: dmd.${{ github.base_ref || github.ref_name }}.freebsd-64.tar.xz
+
+    runs-on: ${{ matrix.host }}
 
     env:
-      OS: ${{ matrix.target.os }}
-      BUILD: ${{ matrix.target.build }}
+      OS: ${{ matrix.os }}
+      BUILD: ${{ matrix.build }}
```

Comment:
So... This was plain wrong before ?

Reply:
It was fine, just always built on master.